pints/_error_measures.py

```diff
@@ -272,6 +272,35 @@ def __call__(self, x):
             (self._problem.evaluate(x) - self._values)**2))
 
 
+class NormalisedRootMeanSquaredError(ProblemErrorMeasure):
```

Comment:
Again, alphabetical!

The reason is, when there's only a few classes you can make a sensible order, but then when there's more it becomes less clear and different people have different opinions so it's easier to just do everything alphabetical. Also makes it easier to find stuff :-)

Reply:
You kept answering my questions above! :-D

Reply:
I have moved this one up too. But there're a few classes that others inherent, they can't go in alphabetical order. So I have left the others stay unchanged.